website/docs/r/organization_token.html.markdown

```diff
@@ -0,0 +1,33 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_organization_token"
+sidebar_current: "docs-tfe-organization-token"
+description: |-
+Generates a new organization token, replacing any existing token.
+---
+
+# tfe_organization_token
+
+Generates a new organization token, replacing any existing token. This token
+can be used to act as the organization service account.
```

Comment:
should the resource for this error if an existing token exists?

Reply:
Good question... Following the API I would say no. But I can see why this could cause unexpected situations. I'll discuss this within the TFE team first.

Reply:
Discussed this with the team... I will add a `force-recreate` option. If that is set to `true` we just warn an recreate, else we give an error!

Good catch! Fixed in next commit.